articles/definitions-time/index.zh-hans.html

```diff
@@ -0,0 +1,218 @@
+  <p>UTC的时区偏移量为0，在<a href="#field_based_time" class="termref">展示时间格式</a>中UTC通常用<code class="kw" translate="no">Z</code>标识。</p>
+  </section>
+
+
+  <section id="field_based_time">
+  <h2>展示时间格式</h2>
+  <p>当使用<dfn id="def_field_based_format">展示时间格式</dfn>书写时间时，需要将日期与时间划分为独立的字段，可以分为年、月、日、时、分、秒等，比如：<samp>2016-09-11T06:10:32</samp>。展示时间格式相较于其他格式，易读性会好很多，比如：<samp>1465621816590</samp> 也表示了同样的时间。</p>
+  <p>展示时间格式于<a href="#utc" class="termref">UTC</a>时间或者本地时间并没有强关联性，与特定的日历（例如公历）有一定的关联。 </p>
+  <p>ISO 8601标准用于描述展示时间格式。</p>
+  </section>
+
+
+  <section id="incremental_time">
+  <h2>累计时间</h2>
+  <p><dfn id="def_incremental_time">累计时间</dfn>是一种在计算机中标识时间的方式，从特定时间点（称为“纪元”）到当前时间累计的整数单位时间。</p>
+  <p> Java（和其他众多系统）的计时方式为，从 <a href="#utc" class="termref">UTC</a> 1970年1月1日午夜（0点）到当前累计的毫秒数（减去中间所有的闰秒）。不过，其他系统可能使用不同的累计单位和开始纪元。</p>
+  <p>举个例子，在书写本段文字时，我的浏览器中JavaScript的累计时间是<samp>1465621816590</samp>（即2016年6月11日上午6点BST）。</p>
+  <p>多数的编程语言和操作系统内部使用累计时间作为时间值。但用户层面通常不会看到累计时间，一般会转换为<a href="#field_based_time" class="termref">展示时间格式</a>方便人们使用。</p>
+</section>
+
+
+  <section id="local_time">
+  <h2>时钟时间/当地时间</h2>
+  <div class="sidenoteGroup">
+  <p> <dfn id="def_wall_time">时钟时间</dfn>或者<dfn id="def_local_time">本地时间</dfn>是一个时刻，加上时区信息后，就可以映射到累计时间的某个特定值上。通常，它对应着人们可识别的时间，获取方式可以通过查看安装在墙上的时钟与日历。</p>
+  <aside class="sideinfonote">
+  <p class="info">在HTML中，描述为<span class="qterm">local</span>的时间并不是指时钟时间。实际上<code class="kw" translate="no">datetime-local</code>的所描述的是一个<a href="#floating_time" class="termref">浮选时间</a>。</p>
+  </aside></div>
+  <div class="sidenoteGroup">
+  <p>举个例子，在写上一段的时候，我无意看了一眼电脑的显示时间，看到 <samp>6月11日星期六 06:10</samp>。结合时间与<a href="utc" class="termref">UTC</a>关联的相关知识（考虑到当时英国的夏令时，调整了一小时），本地时间可以转换为累计时间 <samp>1465621816590</samp>。</p>
+  <p>同时，本地时间也可以转换为另一个区域的本地时间，假如在旧金山，同一时刻计算机上的显示时间为 <samp>6月10日星期五 22:10</samp>。</p>
+  <aside class="sideinfonote">
+  <p class="info"><dfn id="def_observed_time">观测时间</dfn>是指一种依赖于物理观察来确定时间的方式，可以依赖于太阳，月亮或者星星等物理事物，不过当前基本废用了。观测时间里的中午是指太阳在空中的最高点，但对于仅仅相距几公里的人来说，实际的中午时间可能会有所不同。</p>
+  </aside></div>
+  </section>
+
+
+  <section id="floating_time">
+  <h2>浮选时间</h2>
+  <p>有些时间值并不能映射到<a href="incremental_time" class="termref">累计时间</a>中的某个特定时刻。与之相反，它们仅仅代表一个名义时间，在世界各地的所有时区都会以相同的方式呈现。</p>
+  <p>举个例子，2016年6月11日星期六是英国女王官方的90岁生日。由于不同国家的时钟设置差异，6月11日在英国开始或结束的具体时间，在其他国家实际上有可能是周五或者周日了，但该事件的日期始终是6月11日星期六。</p>
+  <p>由于没有对应明确的<a href="#incremental_time" class="termref">累计时间</a>，因此，我们通常称此类时间为<dfn id="def_floating_time">浮选时间</dfn>。当绑定上对应的时区信息后，浮选时间会生成<em>一系列可被接收的累计时间的值</em>。</p>
+  <p>浮选时间不会也不应该归属于某个特定的时区。</p>
+  <p>举一些其他浮选时间事件的例子，比如：报纸的出版日期、里约奥运会的开始日期、新年开始的时间、办公时间设置为“9点到5点”等等，均不需要考虑<a href="#time_zone">时区</a>。</p>
+  </section>
+
+
+<section id="time_zone">
+  <h2>时区</h2>
+  <p><dfn id="def_time_zone">时区</dfn>是一组用于确定本地时间（也叫时钟时间）的规则，它与特定地理区域的累计时间（在大多数的计算系统中使用）相关。</p>
+  <p>如果要关联到时钟时间，时区规则中必须要考虑<a href="#time_zone_offset" class="termref">时区偏移量</a><em>和</em>任何<a href="#dst" class="termref">夏令时</a>的变更。</p>
```

Comment:
```suggestion
  <p>如果要关联到时钟时间，时区规则中必须要考虑<a href="#time_zone_offset" class="termref">时区偏移量</a><strong>和</strong>任何<a href="#dst" class="termref">夏令时</a>的变更。</p>
```

中文传统上不使用斜体引起读者注意，建议改为`<strong>`。